Makefile

```diff
@@ -15,9 +15,7 @@ all: release
 
 .PHONY: release
 release:
-	docker build --no-cache -t aws-fluent-bit-plugins:latest -f Dockerfile.plugins .
 	docker build -t amazon/aws-for-fluent-bit:latest -f Dockerfile .
-	docker build -t amazon/aws-for-fluent-bit:init-latest -f Dockerfile.init .
```

Comment:
for testing only. Will add these lines back.

Reply:
Please remove so I can approve

Reply:
Please remove so I can approve